RELEASE/scripts/autoscend.ash

```diff
@@ -2313,6 +2317,10 @@ boolean L13_towerNSNagamar()
 		return false;
 	}
 	
+	if(creatable_amount($item[Wand Of Nagamar]) == 0 && (creatable_amount($item[WA]) > 0 || item_amount($item[WA]) > 0))
+	{	
+		pullXWhenHaveY($item[ND], 1, 0);
+	}
```

Comment:
I still wouldn't do this. The most common case is you don't have the lowercase n because no one goes to The Valley of Rof L'm Fao in run (unless they're intentionally wasting turns but the script shouldn't at any rate).
The Ruby W comes from one of the Friar Zones, he Metallic A is in the Airship and the Heavy D is from Castle Basement all zones which we have to visit during the run so it's highly likely we will have those but if we're handling one case, we have to handle the other 5 as well.

Reply:
This is 2 lines of actual code (brackets don't count) that save 1 adv on 90%+ of softcore runs. I think that is reasonable.

my original code handles every single possible permutation to save that 1 adventure, but at the cost of being overly large

We could get rid of it entirely, but that is just wasting 1 adventure per softcore run on the majority of runs... seems to go too far.
I don't think handling the most common case obligates us to handle all cases.
... I could specifically check for a case where we are missing only the lowercase n. that probably would only add 2 more lines of code